### DISCUSSION TITLE
Book Author's Name (Editor in Bibtex) for a chapter is not displayed

### DISCUSSION THREAD

**Discussion by antoinelemor** [alshedivat/al-folio#2546] [Q&A]
Hi,

I can't manage to display the book author's name in my publications (Editor entry in bibtex). My Bibtex file is structured as follow, as expected for a bibtex file. Still, the name of the authors of the book are not displayed. 

Can somebody help?

@incollection{lemor_listening_2022,
	title = {Listening to science in policy design: the contrasting cases of {Quebec} and {Sweden} during the early days of the {Covid}-19 pandemic},
	isbn = {978-1-83910-660-6},
	shorttitle = {Listening to science in policy design},
	abstract = {The COVID-19 pandemic has given science, scientific information, and health experts a prominent role in public policy design. However, political decisions are rarely automatically derived from scientific information or expertise. Political decisions are made of complex set of choices, balancing preferences and objectives that are sometimes difficult to reconcile. Moreover, all scientific information entails a degree of uncertainty, especially when dealing with a new problem like COVID-19. In turn, this uncertainty can be a source of discomfort when information is needed to inform a public policy decision. Finally, the interpretation of scientific information can be influenced by elements of contexts, institutional structures, and decision makers' own risk assessment and tolerance. Thus, similar information can be perceived and used differently by policy designers. In this chapter, we focus on the role of information selection and processing in policy design by studying school closure decisions during the early days of the COVID-19 pandemic. Faced with similar infection curves, Sweden and Quebec have made radically different decisions about school closures. While on March 1st 2020 Sweden decided to keep its schools open, on March 13th, Quebec decided to close them down. By drawing from public statements in the two jurisdictions between January and June 2021, we argue that the Swedish authorities have favoured information from the World Health Organization, whereas Quebec has relied more on foreign experiences. The interpretation of this different information and the attitudes towards risk and uncertainty have led the Swedish and Quebec governments to make radically different choices.},
	language = {en\_US},
	chapter = 10,
	urldate = {2023-09-13},
	booktitle = {Research {Handbook} of {Policy} {Design}},
	publisher = {Edward Elgar Publishing},
	author = {Lemor, Antoine and Beaulieu-Guay, Louis-Robert and Öberg, PerOla and Montpetit, Éric},
	editor = {Peters, B. Guy and Fontaine, Guillaume},
	month = apr,
	year = {2022},
	doi = {10.4337/9781839106606.00019},
	pages = {176--193},
}

@incollection{montpetit_forthcoming_2024,
	edition = {Presses de l'Université de Montréal},
	title = {({Forthcoming}) {Science}, jugement humain et décisions de politiques publiques en temps de {COVID}-19},
	booktitle = {Faire preuve : comment nos sociétés distinguent le vrai du faux},
	author = {Montpetit, Éric and Lemor, Antoine},
	editor = {Prud'homme, Julien and Kao, Molly},
	year = {2024},
}

**Comment by george-gca** [comment]
What exactly is being displayed? Please add a link to your site so we can see it, or a screenshot of it. Also, when pasting code or something like that, paste it between 3 backticks identifying the language when possible, in this case bib, so it is easier to read:

````markdown
```bib
@incollection{montpetit_forthcoming_2024,
edition = {Presses de l'Université de Montréal},
title = {({Forthcoming}) {Science}, jugement humain et décisions de politiques publiques en temps de {COVID}-19},
booktitle = {Faire preuve : comment nos sociétés distinguent le vrai du faux},
author = {Montpetit, Éric and Lemor, Antoine},
editor = {Prud'homme, Julien and Kao, Molly},
year = {2024},
}
```
````

which generates

```bib
@incollection{montpetit_forthcoming_2024,
edition = {Presses de l'Université de Montréal},
title = {({Forthcoming}) {Science}, jugement humain et décisions de politiques publiques en temps de {COVID}-19},
booktitle = {Faire preuve : comment nos sociétés distinguent le vrai du faux},
author = {Montpetit, Éric and Lemor, Antoine},
editor = {Prud'homme, Julien and Kao, Molly},
year = {2024},
}
```
